e2e/coverage-handlebars/package.json

```diff
@@ -6,7 +6,7 @@
     "testEnvironment": "node"
   },
   "dependencies": {
-    "handlebars": "^4.7.3",
-    "source-map": "^0.6.1"
+    "@cspotcode/source-map": "0.8.0",
```

Comment:
need to update https://github.com/facebook/jest/blob/75c7c401d4555127c9cee3e27770978a1172e392/e2e/coverage-handlebars/transform-handlebars.js#L9